modules/bookclass.js

```diff
@@ -0,0 +1,36 @@
+/* eslint max-classes-per-file: ["error", 2] */
+
+const bookContainer = document.querySelector('.books');
+export const renderElements = (arr, container) => {
+  container.innerHTML = '';
+  arr.forEach((element, index) => {
+    container.innerHTML += `
+          <div class="book-element">
+            <h2>${element.title}</h2>
+            <p>${element.author}</p>
+            <button type="button" data-id=${index} class="btn-rm"> Remove </button>
+          </div>`;
+  });
+};
```

Comment:
- [ ] To improve the modularity of your project, let's move this function to a separate file and import it here. This will make this file `bookClass.js` to contain only `BookList` class and make your project more modular.